### PR TITLE
coder: 0.12.5 -> 0.13.1

### PR DESCRIPTION
**Pull request by ghuntley**
###### Description of changes

1. bump version
2. add @ghuntley as maintainer

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/coder/default.nix
Comment (ghuntley) [inline pkgs/development/tools/coder/default.nix:32]:
```suggestion
    description = "Coder provisions software development environments via Terraform on Linux, macOS, Windows, X86, ARM, and of course, Kubernetes.";
```

File: pkgs/development/tools/coder/default.nix
Comment (ghuntley) [inline pkgs/development/tools/coder/default.nix:8]: TODO: resolve chucksums.

Comment (urandom2) [reply pkgs/development/tools/coder/default.nix:8]:
```diff
index e80fd5d3518..f14f2440d43 100644
--- a/pkgs/development/tools/coder/default.nix
+++ b/pkgs/development/tools/coder/default.nix
@@ -11,13 +11,13 @@ buildGoModule rec {
     owner = pname;
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-tPpWj2MV2LLIOGq+RTpHpLozgqv7gBgYD3jjehRXOvk=";
+    hash = "sha256-85eH3gKn1khMiksA/kL+fHt1WiaOzVntYCiL1PR/0Cg=";
   };

   # integration tests require network access
   doCheck = false;

-  vendorHash = "sha256-3SStGCDpo+AS4PM9mbXM0EjsJ/3CVFQyb/NRK9RSZ3A=";
+  vendorHash = "sha256-tdqqGM2b8un4BFtvRJsmiIGdb1AOKP8XxcgGg2DilXA=";

   nativeBuildInputs = [ installShellFiles ];

```

Comment (ghuntley) [reply pkgs/development/tools/coder/default.nix:8]: Thanks!

**Review by urandom2 — APPROVED** [review]
if ofborg is happy, so am I

**Comment by ghuntley** [comment]
> if ofborg is happy, so am I

and I'm happy to relieve the maintenance burden from you @urandom2. You may see some extra PR's from other folks who work at Coder in the next couple days 🚀 

CleanShot 2022-12-04 at 11 55 57@2x

**Comment by SuperSandro2000** [comment]
The screenshot is wrong. You cannot merge PRs by being a maintainer but you get some triage functions.

**Comment by ghuntley** [comment]
> if ofborg is happy, so am I

Ofborg is happy. Can I get your approval and let's ship it.

**Review by urandom2 — APPROVED** [review]
I approved a few days back, and that view stands; I do not have merge permissions, we need somebody else to do that, and have already asked in discourse. I think we just have to be patient, sorry.

**Pull request merged**
